### ISSUE TITLE
ASD mixed transports

### ISSUE THREAD

**Issue by khenderick** [openvstorage/alba-asdmanager#105]
ASD mixed transports

**Comment by khenderick** [comment]
Fixed by #106, packaged in openvstorage-sdm-1.6.3-rev.348.c95e34b

**Comment by kinvaris** [comment]
PASSED: `Transport` should be on `tcp` for normal flow between voldrv - proxy - asds; `rora_transport` should be on RDMA.

```
In [2]: Configuration.get("/ovs/alba/asds/ymQk5n9fV1SLcdTxfvbPXOtRknLBMpDG/config")
Out[2]: 
{u'asd_id': u'ymQk5n9fV1SLcdTxfvbPXOtRknLBMpDG',
 u'capacity': 255928844288,
 u'home': u'/mnt/alba-asd/zhg6rjOnqjLCQvvK/ymQk5n9fV1SLcdTxfvbPXOtRknLBMpDG',
 u'ips': [u'172.22.1.30'],
 u'log_level': u'info',
 u'node_id': u'pDXYdN2qskTIvLrarqcFuaNHCd9iTkDw',
 u'port': 8622,
 u'rocksdb_block_cache_size': 62482628,
 u'rora_port': 8623,
 u'rora_transport': u'rdma',
 u'transport': u'tcp'}
```
